examples/end2end_tfkeras_serialize.py

```diff
@@ -0,0 +1,63 @@
+# SPDX-License-Identifier: Apache-2.0
+
+"""
+This example builds a simple model without training.
+It is converted into ONNX directly, by serializing to byte string.
+Predictions are compared to the predictions from tensorflow to check there is no
+discrepencies. Inferencing time is also compared between
+*onnxruntime*, *tensorflow* and *tensorflow.lite*.
+"""
+
+# from https://github.com/microprediction/tensorflow-onnx/blob/master/examples/end2end_tfkeras.py
```

Comment:
I don't think we want this reference to an external repo.

Reply:
shall delete